tests/test_bool/good/test_bool.catala_en

```diff
@@ -39,6 +39,11 @@ $ catala Interpret -s TestBool
 
 ```catala-test-inline
 $ catala Scopelang 
+type TestBool = {
```

Comment:
I would have put `struct` instead of `type` here ; I know this looks like Ocaml but we can afford the extra precision.